README.md

```diff
@@ -1,4 +1,64 @@
 # Ballerina Kafka Module
 
-This module provides support to handle Kafka clients in Ballerina.
-Ballerina Kafka is a part of Ballerina language standard library.
+
+[![Build Master Branch](https://github.com/ballerina-platform/module-ballerinax-kafka/workflows/Ballerina%20Kafka%20Module%20Build/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-kafka/actions?query=workflow%3A%22Ballerina+Kafka+Module+Build%22)
```

Comment:
Rename Github actions to build, Else the badge will be too long

Reply:
Done